carla_integration/bridge.yml

```diff
@@ -84,4 +84,8 @@ topics:
     topic: /hardware_interface/vehicle/twist
     type: geometry_msgs/msg/TwistStamped
     queue_size: 1
+  -
+    topic: /incoming_mobility_path
```

Comment:
Since the bridge.yml files include the full topic names, this should be `/message/incoming_mobility_path` here and in the other files.